src/UIModule.php

```diff
@@ -0,0 +1,202 @@
+<?php
+
+namespace RebelCode\Bookings\WordPress\Module;
+
+use Dhii\Data\Container\ContainerFactoryInterface;
+use Psr\Container\ContainerInterface;
+use Psr\EventManager\EventManagerInterface;
+use RebelCode\Modular\Module\AbstractBaseModule;
+
+class UIModule extends AbstractBaseModule
+{
+    private $bookingsPageId;
+
+    /**
+     * Constructor.
+     *
+     * @since [*next-version*]
+     *
+     * @param ContainerFactoryInterface $containerFactory The factory for creating container instances.
+     * @throws \Dhii\Exception\InternalException
+     */
+    public function __construct(ContainerFactoryInterface $containerFactory)
+    {
+        $this->_initModule(
+            $containerFactory,
+            'bookings-ui-module',
+            ['wp-events'],
+            $this->_loadPhpConfigFile(__DIR__ . '/../config.php')

```

Comment:
I'd rather avoid using relative paths that traverse upwards in the file system. I recommend adding a constant in the `module.php` file that records the module's directory, such as

```
define('WP_BOOKINGS_UI_MODULE_DIR', __DIR__);
```

Then use this constant here.